cmd/root.go

```diff
@@ -115,6 +130,9 @@ func init() {
 	RootCmd.PersistentFlags().BoolP("debug", "d", false, "Activate the debug mode wich includes TF logs via TF_LOG=TRACE|DEBUG|INFO|WARN|ERROR configuration https://www.terraform.io/docs/internals/debugging.html")
 	_ = viper.BindPFlag("debug", RootCmd.PersistentFlags().Lookup("debug"))
 
+	RootCmd.PersistentFlags().String("log-file", path.Join(xdg.CacheHome, "terracognita", "log.log"), "Write the logs with -v to this destination")
```

Comment:
I'd simply rename it to `terracognita.log` or `log(s)/terracognita.log` seems more standard.

Reply:
I'll use `terracognita.log` as we do not have anything else there.